gulpfile.js

```diff
@@ -12,5 +12,8 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
-    mix.less('app.less');
+    mix.less([
+        'app.less',
+        'animate.less'
```

Comment:
animate.less should be added as @import in app.less, modifying gulpfile is not required.